stetho/src/main/java/com/facebook/stetho/json/ObjectMapper.java

```diff
@@ -273,6 +273,18 @@ private Object getJsonValue(Object value, Class<?> clazz, Field field)
       return convertValue(value, JSONObject.class);
     }
     // hmm we should be able to directly serialize here...
```

Comment:
Yes. I'll change it.
On Oct 4, 2015 5:28 PM, "Josh Guilfoyle" <notifications@github.com> wrote:

> In stetho/src/main/java/com/facebook/stetho/json/ObjectMapper.java
> <https://github.com/facebook/stetho/pull/292#discussion_r41098631>:
>
> > @@ -273,6 +273,18 @@ private Object getJsonValue(Object value, Class<?> clazz, Field field)
> >        return convertValue(value, JSONObject.class);
> >      }
> >      // hmm we should be able to directly serialize here...
>
> yes sorry, I meant this comment. it should be just before the last line's
> return statement.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/stetho/pull/292/files#r41098631>.
>